config/application.rb

```diff
@@ -24,7 +24,7 @@ class Application < Rails::Application
     ]
     config.lasso_run = "run"
     config.lasso_commands = "nil"
-    config.lasso_commands_file = '/tmp/delete_resources_steps'
+    config.lasso_commands_file = '/var/lib/delete_resources_steps'
```

Comment:
It's rather unusual to store a file directly in `/var/lib`. I suggest using a sub directory that is owned by the package.